front/app/component-library/components/Button/index.tsx

```diff
@@ -129,16 +125,11 @@ function getButtonStyle(
       textColor: get(props.theme.colors, 'tenantText'),
       textHoverColor: get(props.theme.colors, 'tenantText'),
     },
-    secondary: {
-      bgColor: colors.grey200,
-      textColor: darken(0.1, colors.textSecondary),
-      bgHoverColor: darken(0.05, colors.grey200),
-    },
     'secondary-outlined': {
       bgColor: 'transparent',
       bgHoverColor: transparentize(0.95, colors.textSecondary),
       textColor: colors.textSecondary,
-      borderColor: lighten(0.25, colors.textSecondary),
```

Comment:
We could still use `lighten` btw, just a bit less so this borderColor is 3:1 against a white background.